src/timeline/components/DeleteSessionButton.jsx

```diff
@@ -1,18 +1,29 @@
 const React = require("react");
-require("./navbarStyles.css");
+const { ipcRenderer } = require("electron");
+
+const { useSessionMetrics } = require("./SessionMetricsContext");
+
 
 const DeleteSessionButton = () => {
+  const { sessionId } = useSessionMetrics();
+  const deleteSession = () => {
```

Comment:
You can set up redirecting after deletion by moving this delete button inside PrevSessionProvider in Timeline.jsx
```
            <PrevSessionProvider>
              <PrevSessionsMenu setName={setName}/>
              <NewSessionButton/>
              <DeleteSessionButton/>
            </PrevSessionProvider>

```

and then adding adding this to the DeleteSessionButton
```
const { sessionId, setSessionId } = useSessionMetrics();
const { prevSessionIds } = usePrevSession()
const navigation = useNavigate();

const deleteSession () => {
    ipcRenderer.send("deleteSession", sessionId);
    if(prevSessionIds.length !== 0) {
        setSessionId(prevSessionIds[prevSessionIds.length - 1]); //Load the last session in the list
    }
    else {
        navigation("/"); //Go to home screen if there are no other sessions
    }
}
```

Reply:
I'm confused on how to alter the DeleteSessionButton.jsx code now. Should the 2nd markdown code be within the DeleteSessionButton const that exists already or should i scrap that and just have this as the sole const?